mars/tensor/base/tests/test_base_execute.py

```diff
@@ -444,6 +444,13 @@ def testArraySplitExecution(self):
         [np.testing.assert_equal(r, e) for r, e in zip(res, expected)]
 
     def testSplitExecution(self):
+        for a in ((1,1,1,2,2,3), [1,1,1,2,2,3]):
+            splits = split(a, (3,5))
+            self.assertEqual(len(splits), 3)
+            np.testing.assert_array_equal(splits[0].execute(), (1,1,1))
```

Comment:
`.execute()` is sort of a high-level API, in our tests, we use the code like

```python
self.executor.execute_tensor(splits[0], concat=True)[0]
```

Could you please modify the code again for line 450-452. Thank you so so much.